ext/MitsubaLoader/CElementBSDF.cpp

```diff
@@ -68,29 +68,29 @@ CElementFactory::return_type CElementFactory::createElement<CElementBSDF>(const
 	switch (obj->type)
 	{
 		case CElementBSDF::Type::DIFFUSE:
-			_IRR_FALLTHROUGH;
+			_NBL_FALLTHROUGH;
```

Comment:
actually because we use C++17 now, you can get rid of the FALLTHROUGH macro and just use `[[fallthrough]]`